demos/python_demos/colorization/colorization_demo.py

```diff
@@ -0,0 +1,106 @@
+from openvino.inference_engine import IENetwork, IECore
+import cv2 as cv
+import numpy as np
+import os
+from argparse import ArgumentParser, SUPPRESS
+import xml.etree.ElementTree as ET
+import re
+import logging as log
+import glob
+import sys
+
+
+def build_arg():
+    parser = ArgumentParser(add_help=False)
+    in_args = parser.add_argument_group('Options')
+    in_args.add_argument('-h', '--help', action='help', default=SUPPRESS, help='Help with the script.')
+    in_args.add_argument("-m", "--model", help="Required. Path to .xml file with pre-trained model",
+                         required=True, type=str)
+    in_args.add_argument("-d", "--device",
+                         help="Optional. Specify target device for infer: CPU, GPU, FPGA, HDDL or MYRIAD. "
+                         "Default: CPU",
+                         default="CPU", type=str)
+    in_args.add_argument("-i", "--input", help="Required. Path to a folder with image files", required=True, type=str)
+    return parser
+
+
+def get_parameters(xml_path):
+    root = ET.parse(xml_path).getroot()
+    pattern = r'data\[([0-9]*\.*[0-9]*)\]'
+    scale_data = root.findall("./meta_data/cli_parameters/scale_values")[0]
+    mean_data = root.findall("./meta_data/cli_parameters/mean_values")[0]
+    scale_const = float(re.match(pattern, scale_data.attrib['value']).group(1))
+    mean_const = float(re.match(pattern, mean_data.attrib['value']).group(1))
+    return scale_const, mean_const
+
+
+def get_model_files(args_in):
+    config_xml_in = args_in.model
+    model_path = os.path.splitext(config_xml_in)[0]
+    weights_bin_in = model_path + ".bin"
+    level_up = os.path.split(os.path.split(model_path)[0])[0]
+    coeffs_in = level_up + "/" + os.path.split(level_up)[1] + ".npy"
+    return config_xml_in, weights_bin_in, coeffs_in
+
+
+if __name__ == '__main__':
+    log.basicConfig(format="[ %(levelname)s ] %(message)s", level=log.INFO, stream=sys.stdout)
+
+    args = build_arg().parse_args()
+    config_xml, weights_bin, coeffs = get_model_files(args)
+    scale, mean = get_parameters(config_xml)
+
+    log.info("Load network")
+    load_net = IENetwork(model=config_xml, weights=weights_bin)
+    exec_net = IECore().load_network(network=load_net, device_name=args.device)
+
+    input_blob = next(iter(load_net.inputs))
+    output_blob = next(iter(load_net.outputs))
+
+    n, c, h_in, w_in = load_net.inputs[input_blob].shape
+    images = np.ndarray(shape=(n, c, h_in, w_in))
+
+    log.info("Preparing images data")
+    list_path_images = glob.glob(args.input + "/*")
+
+    for path_image in list_path_images:
+        log.info("#############################")
+        original_frame = cv.imread(path_image)
```

Comment:
There should be a check that the image was read successfully.

Reply:
@Wovchena changed